org.eclipse.xtext/xtend-gen/org/eclipse/xtext/generator/trace/node/WhiteSpaceConfig.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2017 TypeFox GmbH (http://www.typefox.io) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.xtext.generator.trace.node;
+
+/**
+ * @author Sven Efftinge - Initial contribution and API
+ */
+@SuppressWarnings("all")
+public class WhiteSpaceConfig {
```

Comment:
Let's use a name that includes to word `Generator` or similar to avoid confusion with `org.eclipse.xtext.formatting.IWhitespaceInformationProvider`
`org.eclipse.xtext.formatting.IIndentationInformation` and 
`org.eclipse.xtext.formatting.ILineSeparatorInformation`